driverthroughput/main.py

```diff
@@ -53,41 +57,57 @@ def throughput_standard(self,
         :type iterations: int
         :param tensor_bytes: list of tensor sizes in bytes
         :type tensor_bytes: list[int], optional
+        :param languages: which language to use for the tester "cpp" or "fortran"
+        :type languages: str
+        :param wall_time: allotted time for database launcher to run
+        :type wall_time: str
         """
         logger.info("Starting throughput scaling tests")
         logger.info(f"Running with database backend: {get_db_backend()}")
         logger.info(f"Running with launcher: {launcher}")
 
-        exp = create_folder(exp_name, launcher)
-
-        for db_node_count in db_nodes:
-
+        exp, result_path = create_folder(exp_name, launcher)
+        write_run_config(result_path,
+                    colocated=0,
+                    client_per_node=clients_per_node,
+                    client_nodes=client_nodes,
+                    database_nodes=db_nodes,
+                    database_cpus=db_cpus,
+                    iterations=iterations,
+                    tensor_bytes=tensor_bytes,
+                    language=languages,
+                    wall_time=wall_time)
+        permss = list(product(db_nodes, db_cpus))
+        for permsss in permss:
```

Comment:
can you pick a better variable name than `permsss` please?

Reply:
yes MERP nice catch